### DISCUSSION TITLE
Discontinuity of global Hs output at 180° longitude

### DISCUSSION THREAD

**Discussion by Liu-Jincan** [NOAA-EMC/WW3#1186] [Q&A]
Hi, everyone. The below figure is the WW3 global output of the Hs variable, and it's noticeable that there is discontinuous at the 180° longitide.
![IMG_20240204_083357.png](https://github.com/NOAA-EMC/WW3/assets/73934823/0a403df5-49ff-4193-a2bd-871f70da8eb9)

The global grid was generated through the Matlab code of GRIDGEN, and some configurations are as follows:
![Screenshot_2024-02-03-23-50-10-853-edit_org.qpython.qpy.jpg](https://github.com/NOAA-EMC/WW3/assets/73934823/72d52d0a-16ad-499b-a3a1-d3df328841e0)
![Screenshot_2024-02-03-23-50-59-289-edit_org.qpython.qpy.jpg](https://github.com/NOAA-EMC/WW3/assets/73934823/f064813d-95e5-492d-9af2-86c7d50c04f0)

Additionally, here are some settings in the ww3_grid.nml:
![IMG_20240204_085059.jpg](https://github.com/NOAA-EMC/WW3/assets/73934823/f887139e-549f-4263-be1e-a3ff67fb5cf8)

Any suggestion is welcome.

**Comment by aliabdolali** [comment]
You need to see the closure in ww3_grid.inp(nml), and set the flag to T
